### PR TITLE
Week 3 server/step 4

### PR DESCRIPTION
**Pull request by ernestognw**
Comments functionality was completed to handle in-memory comments

The post method to insert comments was added as requested in step 4.

This was accomplished by creating the doPost method in servlet to handle comment POSTING from front. Also, the layout added in #5 was used and enabled correctly.

**Note**: Please review and approve #5 first

### CODE REVIEW

File: portfolio/src/main/webapp/scripts/api/index.js [hunk 1]
```diff
@@ -1,9 +1,44 @@
 import { api } from './client.js';
+import { getCommentTemplate } from './utils.js';
 
-const addQuote = async () => {
-  const quote = await api.get('/data');
+const updateComments = async () => {
```
Comment (livvielin) [inline portfolio/src/main/webapp/scripts/api/index.js:4]: Let's add comments on the methods here and other files: https://google.github.io/styleguide/jsguide.html#jsdoc

File: portfolio/src/main/java/com/google/sps/servlets/CommentsController.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+public class CommentsController extends HttpServlet {
+  private List<Comment> comments;
+
+  @Override
+  public void init() {
+    comments = new ArrayList<>();
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (livvielin) [inline portfolio/src/main/java/com/google/sps/servlets/CommentsController.java:42]: Please add Javadoc: https://google.github.io/styleguide/javaguide.html#s7-javadoc

File: portfolio/src/main/webapp/scripts/api/index.js (hunk 1 above)
Comment (ernestognw) [reply portfolio/src/main/webapp/scripts/api/index.js:4]: Great! Let me work on that

**Comment by ernestognw** [comment]
As said in today's daily, I'll be adding documentation on comments in a future PR

**Pull request merged**